### DISCUSSION TITLE
A comprehensive PIML benchmark tool uses DeepXDE for PINNs

### DISCUSSION THREAD

**Discussion by praksharma** [lululxvi/deepxde#1004] [General]
This PIML benchmark tool uses DeepXDE for PINNs. 

https://openreview.net/pdf?id=dh_MkX0QfrK

**Comment by lululxvi** [comment]
They compare PINN with other methods like U-Net, which doesn't make too much sense.

**Reply by praksharma** [reply]
I agree. U-net needs the ground truth and takes long time to train. PINN is "use and throw" stuff with crazy speed compared to its contemporary ML models for solving PDEs.
